Makefile

```diff
@@ -403,6 +404,18 @@ CRDOC = $(shell pwd)/bin/crdoc
 crdoc: ## Download crdoc locally if necessary.
 	$(call go-get-tool,$(CRDOC), fybrik.io/crdoc,v0.5.2)
 
+.PHONY: chainsaw
+chainsaw: ## Find or download chainsaw
+ifeq (, $(shell which chainsaw))
+	@{ \
+	set -e ;\
+	go install github.com/kyverno/chainsaw@v0.1.4 ;\
+	}
+CHAINSAW ?= $(GOBIN)/chainsaw
```

Comment:
If we use the method suggested above, then this can simply be `$(LOCALBIN)/chainsaw`, and this whole conditional may be simplified, as we already detect whether chainsaw is installed globally before this point.

Reply:
You are probably right, i didn't spend much time on this part. Will look at it in more depth after lunch (or in a follow up if it's ok).

Reply:
Follow up is fine as well, e2e tests are modified often and it's not very practical to keep this PR blocked for minor issues.